CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Splunk Enterprise SDK for Python Changelog
 
+## Version 1.6.18
+
+### Bug fixes
+* [#405](https://github.com/splunk/splunk-sdk-python/pull/405) Fix searchcommands_app example
+* [#406](https://github.com/splunk/splunk-sdk-python/pull/406) Fix mod inputs examples
+* [#407](https://github.com/splunk/splunk-sdk-python/pull/407) Modified Streaming and Generating Custom Search Command
```

Comment:
This probably needs edits to make it more clear what was fixed - `Modified Streaming and Generating Custom Search Command` probably wasn't a good title for that PR. Can we change this to:

> Fixed issue with Streaming and Generating Custom Search Commands dropping fields that aren't present in the first row of results. More details on how to opt-in to this fix can be found here: https://github.com/splunk/splunk-sdk-python/blob/develop/README.md#customization

Reply:
Yes, this makes more sense. I'll make the change.